src/routes/UserRoutes.ts

```diff
@@ -21,6 +21,8 @@ export class UserRoutes {
 
 		router.get('/users/:id', getUser(TokenType.Auth), isVerified, this.userController.getUser.bind(this.userController));
 
+		router.get('/users/:id/report', getUser(TokenType.Auth), isVerified, this.userController.reportUser.bind(this.userController));
```

Comment:
```suggestion
		router.post('/users/:id/report', getUser(TokenType.Auth), isVerified, this.userController.reportUser.bind(this.userController));
```

`POST` is a HTTP verb that indicates you're creating a resource, so it's better suited for this route.